app/api/event/config/routes.json

```diff
@@ -0,0 +1,140 @@
+{
+  "routes": [
+    {
+      "method": "GET",
+      "path": "/events/find",
```

Comment:
@aramkarapetian Do we really want to have _cf-events*_ and _events*_ apis separated but in the same repository?